Add missing headers

tools/legacy/sample_common/src/vaapi_utils.cpp:29:28: error: use of undeclared identifier 'basename'
        so_handle = dlopen(basename(name), RTLD_GLOBAL | RTLD_NOW);
                           ^

--- tools/legacy/sample_common/src/vaapi_utils.cpp.orig	2021-09-13 22:51:02 UTC
+++ tools/legacy/sample_common/src/vaapi_utils.cpp
@@ -8,6 +8,7 @@
 
     #include "vaapi_utils.h"
     #include <dlfcn.h>
+    #include <libgen.h>
     #include <stdexcept>
 
     //#if defined(LIBVA_DRM_SUPPORT)
